src/schemas/raster.schema.json

```diff
@@ -4,22 +4,71 @@
   "title": "Vitessce image data",
   "type": "object",
   "definitions": {
-    "raster": {
+    "imageTransform": {
       "type": "object",
-      "additionalProperties": true,
-      "required": ["sample", "tileSource"],
+      "additionalProperties": false,
+      "required": ["scale", "translate"],
       "properties": {
-        "sample": { "type": "number" },
-        "tileSource": {
-          "oneOf": [
-            { "type": "object" },
-            { "type": "string" }
-          ]
+        "scale": { "type": "integer" },
+        "translate": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["y", "x"],
+          "properties": {
+            "y": { "type": "integer" },
+            "x": { "type": "integer" }
+          }
         }
       }
+    },
+    "imageDimensions": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["field", "type", "values"],
+        "properties": {
+          "field": { "type": "string" },
+          "type": {
+            "type": "string",
+            "examples": ["quantitative", "nominal", "ordinal", "temporal"]
+          },
+          "values": {
+            "type": ["array", "null"],
+            "items": { "type": "string" }
+          }
+        }
+      }
+    },
+    "imageMetadata": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["dimensions", "is_pyramid", "transform"],
+      "properties": {
+        "dimensions": { "$ref": "#/definitions/imageDimensions" },
+        "is_pyramid": { "type": "boolean" },
+        "transform": { "$ref": "#/definitions/imageTransform" }
+      }
+    },
+    "image": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["name", "url", "type", "metadata"],
```

Comment:
Not sure there is an action item here, but I think `metadata` will be optional (might change when I integrate CODEX) since I anticipate OMEXML capturing most of that for pyramids/CODEX.

Reply:
Yeah definitely good to have on our radar, but I think right now the machinery in vitessce is set up expecting these metadata. In the next cycle, I think vitessce should call infer these from the loaders after initialized:

```javascript
const { dimensions } = loader;
```

instead of:

```javascript
const { metadata } = raster_json;
const { dimensions } = metadata;
```